data/hai/hai reveal 5 philosophers.txt

```diff
@@ -686,13 +697,37 @@ mission "Hai Reveal [B09] Scanning Devil-Hide"
+						has "Hai Reveal [B09] Scanning Devil-Hide: label moon"
+			label planet
+			action
+				set "Hai Reveal [B09] Scanning Devil-Hide: label planet"
+			`	"Okay, the planet..." Her voice trails off. "Wow! That's odd. This scanner is definitely not designed to properly scan this thing; I can only get surface readings."`
 			`	"Whatever it is, this planet is not normal by any stretch. It is completely inhospitable and appears to have a huge network of caves and caverns with crystalline structures that are radiating... something. Whatever it is, it's messing with the scanner's attempts to identify it. I'd have to know more about this scanner's functions to guess what the interference is. There's deuterium though maybe denizens of this system siphon the deuterium to make hyperspace fuel? Doesn't seem very efficient, but the solar wind is thin here. There's some kind of gravitational anomaly though, or maybe its just some kind of interference lensing. It looks like the planet has about a thousand times the mass it should, but with gravity around Earth normal. That... can't be right."`
-			`	She shakes it off and looks at you. "Whatever it is we're looking for its not down there. Let me look at the moon... hmmm... okay. It's just an ordinary chunk of rock, but the planet's radiation is disturbing the sensors. I can only see a few dozen meters below ground even though the scanner should be able to look down at least ten times that far... Ahah!"`
+			`	She shakes it off and looks at you. "Whatever it is we're looking for its not down there."`
+				goto scan
+			label moon
+			action
+				set "Hai Reveal [B09] Scanning Devil-Hide: label moon"
+			`	The moon... hmmm... okay. It's just an ordinary chunk of rock, but radiation from the planet is disturbing the sensors. I can only see a few dozen meters below ground even though the scanner should be able to look down at least ten times that far... Ahah!"`
 			`	"I found something. There's a facility buried in the moon's surface. There is definitely a lot of Hai tech too, so looks like the concerns you told me about were spot on. I can only see the uppermost part of the facility but it appears to be divided into two parts. One section has hydroponics, big cargo bays, docking bays for ships, even a small shipyard. Doesn't look like they have space for more than a handful of large ships though. It also seems to have the only decently sized air processor I can identify. The other section is deeper, and appears to have spacious living areas I can see some support systems, but it seems that the air supply is effectively controlled from the surface access section. I can detect 1,387 people, though no details about them, and that's just in the upper three floors that I can see."`
 			`	She looks at you eyes wide. "This isn't just a base; there's a base down there but it's controlling a whole colony."`
 			`	There is a pregnant pause for a moment and then she suddenly gasps. "Captain, this must be Elenchus!"`
```

Comment:
```suggestion
			`	She looks at you, eyes wide. "This isn't just a base; there's a base down there, but it's controlling a whole colony."`
			`	There is a meaningful pause for a moment and then she suddenly gasps. "Captain, this must be Elenchus!"`
```

Reply:
The word "meaningful" does not have the same meaning as "pregnant" (even in the non-reproductive sense of the word). It is not a suitable replacement.

Reply:
'Meaningful' means both long and importance here, that was what pregnant meant here. It was the closest synonym I could find.

Reply:
It doesn't express the type of meaning, though. Why do we need to give the pause an abortion? Isn't "pregnant" a sufficient adjective?

I put in your two commas, though, in another commit just now.